code/modules/projectiles/guns/projectile/toy.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/weapon/gun/projectile/automatic/toy

```

Comment:
should really probably be it's own subtype of /obj/item/weapon/gun/projectile/automatic/toy